flowmanager/datasets.py

```diff
@@ -16,22 +16,22 @@
 
 SCHEMA = {
     'fields': [
-        {'name': 'id', 'type': 'string'},
-        {'name': 'name', 'type': 'string'},
-        {'name': 'title', 'type': 'string'},
-        {'name': 'description', 'type': 'string'},
+        {'name': 'id', 'type': 'string', 'analyzer': 'keyword'},
```

Comment:
this won't set the analyzer. You need to override the mapper class here https://github.com/datahq/specstore/pull/29/commits/d3bc547e5c76b07e1573890fd3b27b27e4b326f5#r160045579